### DISCUSSION TITLE
First example usage

### DISCUSSION THREAD

**Discussion by 9p6** [emmercm/igir#914] [Q&A] [answered]
Hello. I don't quite understand the first example usage:
```
igir copy report --dat *.dat --input **/*.zip --input ROMs/ --output ROMs/
```
If the action is to copy roms then why are the input and output directories the same? Also why there are two input options one with files (multiple arguments) and the other with directory?
Thank you

**Comment by emmercm** [comment]
The goal is to show how you can merge some "new" ROMs (`**/*.zip`) into an "existing" collection (`ROMs/`). But I see how this is confusing because it will achieve the same effect if you leave off `--input ROMs/` entirely.

I think a more useful example would be one where `--input ROMs/` is actually consequential, such as when running the `clean` command - so I'll update with the next release.

**Comment by 9p6** [comment]
I see but `**/*.zip` glob will still match the zips that are already inside `ROMs/`? Or is idea that they're being unpacked in the process of copy? Perhaps I need to run it and find out!

**Reply by emmercm** [reply, marked as answer]
I agree the `**/*.zip` example was confusing, the example was updated here: https://github.com/emmercm/igir/blob/e8e5cdb79ae4ed9c1abf7cbd096ab1222f72e95f/src/modules/argumentsParser.ts#L767-L768
